src/DotLiquid/StandardFilters.cs

```diff
@@ -716,11 +734,15 @@ private static object DoMathsOperation(object input, object operand, Func<Expres
             if (input == null || operand == null)
                 return null;
 
+            bool shouldConvertStrings = false;

```

Comment:
Nit: The assignment of the variable, if condition, all can be done on a single line direct assignment.

Reply:
Too many conditions to be readable IMHO.